### PR TITLE
add routing (placeholders)

### PR DESCRIPTION
**Pull request by pateva**

### CODE REVIEW

File: address_book_ui/src/app/app.component.ts
Comment (NataliaZaharieva) [inline address_book_ui/src/app/app.component.ts:1]: IMO next to the 'app' folder you can have 'assets' and there you can store configs, images, fonts etc.

Comment (NataliaZaharieva) [inline address_book_ui/src/app/app.component.ts:1]: IMO another suggestion for the structure is inside the 'app' you can have a 'core' folder with the layout, header and nav-bar as separate components. 
Next to the 'core' folder you can have 'shared' for some reusable components (if needed). 
And next to the 'core' and 'shared' you can store the other components into 'components' and if you want you can have a folder for a whole page - 'pages'

File: address_book_ui/src/app/app.routes.ts
Comment (NataliaZaharieva) [inline address_book_ui/src/app/app.routes.ts:1]: IMO it is better to have a home route with '' path and if you do not have a home landing page you can navigate to the route that holds the first page  - redirectTo: ROUTES.query
*Also you can store your paths into a constant

File: address_book_ui/src/app/app.routes.ts [hunk 1]
```diff
@@ -1,3 +1,16 @@
 import { Routes } from '@angular/router';
+import { RecordComponent } from './record/record.component';
+import { TestComponent } from './test/test.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+    {
+        path: 'test',
+        component: TestComponent,
+        title: "Placeholder 2"
```
Comment (NataliaZaharieva) [inline address_book_ui/src/app/app.routes.ts]: is the title needed for the nav-bar?

**Review by NataliaZaharieva — CHANGES_REQUESTED** [review]

File: address_book_ui/src/app/app.routes.ts (hunk 1 above)
Comment (pateva) [reply address_book_ui/src/app/app.routes.ts]: I am honestly not sure yet, the title was part of the angular tutorial I was reading to setup some routes

**Pull request merged**